### PR TITLE
Enable ha-icons in markdown card

### PR DESCRIPTION
**Pull request by thomasloven**
```yaml
type: markdown
title: ha-icon
content: |
  This is a markdown card! With mdi icons!
  <ha-icon icon="mdi:home-assistant"></ha-icon>

  Right in the <ha-icon icon="mdi:format-align-middle"></ha-icon> of the <ha-icon icon="mdi:text"></ha-icon> too.
```

image

### CODE REVIEW

**Comment by bramkragten** [comment]
I don't think this should be under the `allow-svg` attribute but either always allowed or with a separate attribute. Or we should rename the attribute it if it makes sense to allow or deny the use of svg and ha-icon always at the same time?

**Comment by thomasloven** [comment]
Good point.

**Comment by balloob** [comment]
I don't remember anymore why we don't allow svg all the time.

**Comment by thomasloven** [comment]
I really can't see any reason either. The markdown card is pretty much the only place where ordinary users have access to it, and why shouldn't they be able to draw stuff if they want to?

I think I managed to remove all references to it too.

**Comment by Mariusthvdb** [comment]
not sure if this is still allowed here, so sorry if not, but since your talking about expanding the icon possibilities in HA, please look at adding these options too:
https://community.home-assistant.io/t/customui-discussion-thread/48694/190?u=mariusthvdb

which was suggested at: https://github.com/Templarian/MaterialDesign/issues/3057#issuecomment-397065190

thanks for considering.

File: src/panels/lovelace/cards/hui-markdown-card.ts [hunk 1]
```diff
@@ -54,6 +54,7 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
             "no-header": !this._config.title,
           })}"
           .content="${this._config.content}"
+          allow-svg
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-markdown-card.ts]: Should be removed?

Comment (thomasloven) [reply src/panels/lovelace/cards/hui-markdown-card.ts]: Yes. Found another one as well.

**Review by balloob — APPROVED** [review]

**Pull request merged**